Documentation/install/system_requirements.rst

```diff
@@ -136,6 +147,26 @@ by adding the following to the helm configuration command line:
      ...
      --set global.enableXTSocketFallback=false
 
+Advanced Features and Required Kernel Version
+=============================================
+Cilium requires Linux Kernel 4.9.17 or higher, however development on additional
+kernel features and functionality continues to progress in the Linux community.
+Some Cilium features and functionality are dependent on newer kernel versions.
+These additional Cilium features and functionality are enabled by upgrading to
+a later kernel version as detailed below:
+
+========================== ===============================
+Cilium Feature             Minimum Kernel Version
+========================== ===============================
+`Longest Prefix Match`_    >= 4.11
+`Host Reachable Services`_ >= 4.19.57, >= 5.1.16+,  >=5.2+
+`Kube-proxy Free`_         >= 4.19.57, >= 5.1.16+,  >=5.2+
+========================== ===============================
+
+.. _Longest Prefix Match: |SCM_WEB|\/install/upgrade/#restrictions-on-unique-prefix-lengths-for-cidr-policy-rules
```

Comment:
When available we also use LRU (Least Recently Used) hash maps with BPF, which appeared [in kernel 4.10](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=29ba732acbeece1e34c68483d1ec1f3720fa1bb3). But as far as I know we don't have any specific documentation on the topic (not sure it makes a difference to the user, other than for performance), so not sure if worth adding in this table.

Reply:
I think for current usage (existing maps) we work around the lack of LRU by just creating a hashmap and spinning up a garbage collection controller in the cilium-agent.